modele-social/règles/salarié.yaml

```
@@ -159,43 +161,152 @@ contrat salarié . frais professionnels . titres-restaurant . contrôle taux emp
 
-contrat salarié . frais professionnels . indemnité kilométrique vélo . distance mensuelle:
-  question: >-
-    Quelle est la distance parcourue en vélo chaque mois pour le trajet domicile / travail ?
-  suggestions:
-    2 km/jour: 40 km/mois
-    5 km/jour: 100 km/mois
-    10 km/jour: 200 km/mois
-  par défaut: 80 km/mois
+contrat salarié . frais professionnels . transports personnels . forfait mobilités durables:
+  valeur: oui
+
+contrat salarié . frais professionnels . transports personnels . forfait mobilités durables . montant:
+  titre: Prise en charge des frais de transports forfait mobilités durables
+  question: Quel montant l'employeur prend-il en charge dans le cadre du forfait mobilités durables ?
+  unité: €/mois
+  par défaut: 0 €/mois
+  description: |
+    L'employeur peut prendre en charge tout ou partie des frais de déplacement liés à l'utilisation des véhicules entrant dans le cadre du forfait mobilités durables
+
+      - le vélo et vélo à assistance électrique
+
+      - le covoiturage (conducteur ou passager)
+
+      - les engins de déplacement personnels en location ou en libre-service
+
+      - l'autopartage avec des véhicules électriques, hybrides rechargeables ou hydrogènes
+
+      - les transports en commun (hors abonnement).
+
+    L'ancienne Indemnité Kilométrique Vélo entre maintenant dans ce dispositif. Elle peut être poursuivie mais son montant devra être imputé ici.
+
+    L'employeur peut prendre en charge ces frais jusqu'à 500€/mois de manière exonérée de cotisations sociales et d'impôt. Attention cependant
+
+      - le plafond est réduit du le montant de la prise en charge des frais d'abonnement aux transports publics
+
+      - la prise en charge du carburant faible émission entre dans cette assiette également.
+
+    Dans le cas d'un temps partiel, l'avantage sera le même pour un mi-temps ou plus. En dessous, un facteur proportionnel sera appliqué.
+
+    Pour verser une prime de salaire équivalente à 500€/mois à son salarié sans ce dispositif, **l'employeur devrait débourser près de 800€ pour un salaire médian**.
+  références:
+    Articles R3261-13-1 à -13-2 du code du travail, version 11/05/2020: https://www.legifrance.gouv.fr/codes/section_lc/LEGITEXT000006072050/LEGISCTA000018487476/2020-05-11
+    Article 81 du code des impôts, version en vigueur au 31/12/2020: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000042910732/2020-12-31/
+
+contrat salarié . frais professionnels . transports personnels . forfait mobilités durables . part déductible:
+  titre: Prise en charge des frais de transports forfait mobilités durables (part déductible)
+  unité: €/mois
+  valeur:
+    le maximum de:
+      - 0
+      - le minimum de:
+        - montant
+        - somme:
+          - le maximum de:
+            - 0
+            - proportion déduction * 500€/mois - abonnement transports publics . prise en charge
+          - 0 - carburant faible émission . part déductible
```

Je propose la réecriture suivante qui me semble équivalente : 
```suggestion
  valeur: montant
  plafond:
     valeur: proportion déduction * 500€/mois 
     abattement: 
       somme: 
         - abonnement transports publics . prise en charge
         - carburant faible émission . part déductible
```